src/app/blog/new-article.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+
+import { ArticleService } from './article.service';
+import { Article, ArticleFormData } from './article.type';
+
+@Component({
+  selector: 'bb-new-article',
+  templateUrl: './new-article.component.html',
+})
+export class NewArticleComponent implements OnInit {
+  user = {
+    id: '9c34d115-a13e-43d0-949b-c347a8347fbb',
+    firstName: 'Oralle',
+    lastName: 'Stanlick',
+    avatarUrl: 'https://robohash.org/doloresinducimus.png?size=50x50&set=set1',
+  };
+
+  newArticleData: ArticleFormData = {
+    title: '',
+    description: '',
+    text: '',
+    tags: '',
+  };
```

Comment:
better to use it as creating new model.
Create class User, which accepts all these params. It will be better to create user/article instances via class, it also could be a type, example:
```
user: UserModel = new UserModel({
    id: '9c34d115-a13e-43d0-949b-c347a8347fbb',
    firstName: 'Oralle',
    lastName: 'Stanlick',
    avatarUrl: 'https://robohash.org/doloresinducimus.png?size=50x50&set=set1',
  })
  
 or
 
 user: UserModel = new UserModel('9c34d115-a13e-43d0-949b-c347a8347fbb', 'Oralle','Stanlick','https://robohash.org/doloresinducimus.png?size=50x50&set=set1',
  )
  ```